### PR TITLE
Update calls to Process.Start()

### LINKED ISSUE #4995: Update calls to Process.Start
Unless we're launching the interpreter that the user typed in, we should be making sure to use full paths when launching executable.

Example of one that needs to be updated:
https://github.com/Microsoft/PTVS/blob/1d04f01b7b902a9e1051b4080770b4a27e6e97e7/Python/Product/EnvironmentsList/EnvironmentPathsExtension.xaml.cs#L81

We can use `Path.Combine(Environment.SystemDirectory, "explorer.exe")` to specify the full path.

Alternative: `string filepath= "%WINDIR%\\system32\\explorer.exe"; Environment.ExpandEnvironmentVariables(filepath);`

### PR DESCRIPTION
**Pull request by RaymonGulati1**
Fix #4995 

Update calls to Process.Start()

### CODE REVIEW

File: Python/Product/PythonTools/PythonTools/InterpreterList/InterpreterListToolWindow.cs [hunk 1]
```diff
@@ -523,13 +523,14 @@ class CondaEnvironmentManagerUI : ICondaEnvironmentManagerUI {
...
+            var psi = new ProcessStartInfo(Path.Combine(Environment.SystemDirectory, "cmd.exe")) {
+                Arguments = string.Join(" ", new[] {
                 "/S",
                 "/K",
                 pathCmd + string.Format("title {0} environment", view.Description)
-            }.Select(ProcessOutput.QuoteSingleArgument));
-            psi.WorkingDirectory = view.PrefixPath;
+            }.Select(ProcessOutput.QuoteSingleArgument)),
+                WorkingDirectory = view.PrefixPath
+            };
```
Comment (huguesv) [inline Python/Product/PythonTools/PythonTools/InterpreterList/InterpreterListToolWindow.cs]: This is hard to read with this indentation. Arguments and WorkingDirectory should be the only 2 lines at that indent level.

File: Python/Product/PythonTools/PythonTools/InterpreterList/InterpreterListToolWindow.cs [hunk 2]
```diff
@@ -545,14 +546,15 @@ class CondaEnvironmentManagerUI : ICondaEnvironmentManagerUI {
...
-            psi.WorkingDirectory = view.PrefixPath;
+            var psi = new ProcessStartInfo(Path.Combine(Environment.SystemDirectory, "WindowsPowerShell", "v1.0", "powershell.exe")) {
+                Arguments = string.Join(" ", new[] {
+            "-NoLogo",
+            "-NoExit",
+            "-Command",
+            pathCmd + string.Format("(Get-Host).UI.RawUI.WindowTitle = '{0} environment'", view.Description)
+            }.Select(ProcessOutput.QuoteSingleArgument)),
+                WorkingDirectory = view.PrefixPath
+            };
```
Comment (huguesv) [inline Python/Product/PythonTools/PythonTools/InterpreterList/InterpreterListToolWindow.cs]: Same comment as above. The older code was easier to read.

File: Python/Product/PythonTools/PythonTools/InterpreterList/InterpreterListToolWindow.cs (hunk 1 above)
Comment (RaymonGulati1) [reply Python/Product/PythonTools/PythonTools/InterpreterList/InterpreterListToolWindow.cs]: Updated. Not sure if this is exactly what you wanted, but I feel that it's much easier to read.

File: Python/Product/PythonTools/PythonTools/InterpreterList/InterpreterListToolWindow.cs (hunk 2 above)
Comment (RaymonGulati1) [reply Python/Product/PythonTools/PythonTools/InterpreterList/InterpreterListToolWindow.cs]: Same as above.

**Pull request merged**